### ISSUE TITLE
bash should be added as a dependency, for now

### ISSUE THREAD

**Issue by keevan** [ChaoticWeg/discord.sh#41]
https://github.com/ChaoticWeg/discord.sh/blob/2f2c0f4c14b48474a6a360eb401c040016cfbedb/README.md?plain=1#L32-L36

I've tried running this on `sh` and there were a few bash specific things that stopped this from getting very far, namely 
```
discord.sh: line 6: shopt: not found
discord.sh: line 7: shopt: not found
discord.sh: line 31: syntax error: bad substitution
```
There's probably many more that it hasn't reached yet but it might be easier to set the requirements to include bash.

**Comment by fieu** [comment]
> I've tried running this on `sh` and there were a few bash specific things that stopped this from getting very far...

This project has never mention the support of `sh` so I don't know why you are trying it.

**Comment by keevan** [comment]
> This project has never mentioned supporting `sh` so I don't know why you are trying it.

So the reason I stumbled upon this was because I saw `.sh` in the title of the project (I haven't checked it's origins), and also because I tested it locally (on a machine with bash), tested some options, and looked at the requirements/dependencies. I then wanted to add this to an alpine image (due to it's compact size). After adding it, going through the motions, and finally executing it, it didn't work the way I thought it would. 

Yes, there is no mention of it supporting `sh`, but I thought I'd save time potentially for others like me by listing it explicitly in the dependencies.
